Error with proposal script against pre-release 0.30.0 core node software

```
Logging into wallet: ******
Logged in to wallet successfully
Selected pubkey for signing
Found an asset with symbol tDAI
{'ID': '6d9d35f657589e40ddfb448b7ad4a7463b66efb307527fedd2aa7df1bbd5ea61', 'name': 'Dai (dev)', 'symbol': 'tDAI', 'totalSupply': '4577810003200000', 'decimals': '5', 'source': {'erc20': {'contractAddress': '0xBEaF29DCE23B39FAfa7BA59a1b3Feb8687f96782'}}}
Found governance asset account
{'id': '', 'owner': '66d7bfd5ea74609cd397946ca62eabee4fee89b75f452a8207ff4d25ad3a6a11', 'balance': '1', 'asset': 'f11862be7fc37c47372439f982a8f09912c4f995434120ff43ff51d9c34ef71a', 'marketID': '', 'type': 'ACCOUNT_TYPE_GENERAL'}
Blockchain time: 1611060037681064477 (1611060037 seconds past epoch)
Traceback (most recent call last):
  File "propose-markets/propose-vote-enact-market.py", line 231, in <module>
    helpers.check_response(response)
  File "/home/edd/src/vega/sample-api-scripts/helpers.py", line 8, in check_response
    assert (
AssertionError: https://.../governance/prepare/proposal returned HTTP 400 {
  "code": 3,
  "message": "InvalidArgument error",
  "details": [
    {
      "@type": "type.googleapis.com/vega.ErrorDetail",
      "code": 10015,
      "message": "malformed request",
      "inner": "invalid field Proposal.NewMarket.Changes.Instrument.Future.SettlementAsset: value '' must not be an empty string"
    }
  ]
}
```
I haven't gone to check, but I am guessing there is an upcoming breaking API change.

--- a/propose-markets/propose-vote-enact-market.py
+++ b/propose-markets/propose-vote-enact-market.py
@@ -178,20 +178,21 @@ market = {
         # Set validation timestamp to a valid time offset from the current Vega blockchain time
         "validationTimestamp": blockchain_time_seconds + 1,
         # Set closing timestamp to a valid time offset from the current Vega blockchain time
-        "closingTimestamp": blockchain_time_seconds + 3610,
+        "closingTimestamp": blockchain_time_seconds + 3601,
         # Set enactment timestamp to a valid time offset from the current Vega blockchain time
-        "enactmentTimestamp": blockchain_time_seconds + 3620,
+        "enactmentTimestamp": blockchain_time_seconds + 3701,
         # Note: the timestamps above are specified in seconds, and must meet minimums required by network
         "newMarket": {
             "changes": {
                 "continuous": {"tickSize": "0.01"},
                 "decimalPlaces": "5",
                 "instrument": {
-                    "code": "CRYPTO:BTCDAI/MAR21",
+                    "code": "CRYPTO:BTCDAI/JUN21",
                     "future": {
                         # Settlement asset identifier (found above)
-                        "asset": found_asset_id,
-                        "maturity": "2021-03-31T23:59:59Z",
+                        "settlementAsset": found_asset_id,
+                        "quoteName": "DAI",
+                        "maturity": "2021-06-30T23:59:59Z",
                         # "settlementPriceSource: {
                         #     "sourceType": "signedMessage",
                         #     "sourcePubkeys": ["YOUR_PUBKEY_HERE"],
@@ -204,7 +205,6 @@ market = {
                         # }
                     },
                     "name": "BTC/DAI",
-                    "quoteName": "DAI",
                 },
                 "logNormal": {
                     "params": {"mu": 0, "r": 0.016, "sigma": 0.05},
@@ -212,7 +212,6 @@ market = {
                     "tau": 1.90128526884173e-06,
                 },
                 "metadata": [],
-                "openingAuctionDuration": "120",
                 "priceMonitoringParameters": {
                    "triggers": [{
                      "auctionExtension": "300",

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 googleapis-common-protos==1.52.0
-grpcio==1.33.2
+grpcio==1.35.0
 requests==2.25.1
-Vega-API-client==0.27.0
+Vega-API-client==0.30.0
 websocket-client==0.57.0

